app/workers/asset_manager_update_whitehall_asset_worker.rb

```diff
@@ -0,0 +1,15 @@
+class AssetManagerUpdateWhitehallAssetWorker < WorkerBase
+  sidekiq_options queue: "asset_manager_updater"
+
+  def perform(attachment_data_global_id, attributes)
+    asset_data = GlobalID::Locator.locate(attachment_data_global_id)
+    path = asset_data.file.asset_manager_path
+    AssetManager::AssetUpdater.call(asset_data, path, attributes)
+
+    # Update PDF thumbnail
+    if asset_data.try(:pdf?)
```

Comment:
rather minor but do we want to be checking `asset_data.file.thumbnail` (or maybe `asset_data.file.try(:thumbnail)` for existence? I'm wondering whether there's some edge cases (e.g. slow to generate preview) where you don't have a thumbnail image.

Reply:
~Annoyingly, every `AttachmentData#file` (which returns an `AttachmentUploader`) will respond to `:thumbnail`, even if it doesn't have a thumbnail preview.~

~Thumbnails are [only generated for PDFs](https://github.com/alphagov/whitehall/blob/d5c9121d1bf4109a7ddc72c76417d81c1e81b47f/app/uploaders/attachment_uploader.rb#L26-L41), but the `#thumbnail` method always exists & returns an object regardless.~

You've actually sent me down a helpful path – thanks @kevindew! It turns out `ImageData` images get resized into various different versions – but we were only applying the auth_bypass_id to the main file.

I think there'll be a way for us to update additional CarrierWave file 'versions' so that they get auth_bypass_ids too.

Reply:
@kevindew take a look at 70781e1cab699a9d623fe927775b4bef9c2d872e because I think it'll resolve this comment.